### ISSUE TITLE
Suggestions

### ISSUE THREAD

**Issue by jeremgod** [Mytherin/Tibialyzer#94]
I don't know if it is appropriate to use the Issues section as a place to suggest improvements or not but I do have a few in mind at the moment, relatively easy I believe

First I was wondering if it would be possible to have the health list HUD have a reverse order option. What I mean by this is I like to have the health list near my cooldowns however I would like my EK's name to appear at the bottom and then anyone else in the list would appear in order that they appear in the character health list (as they do now but instead of stacking top to bottom the would stack bottom to top therefore my EK would always be the first one above the cooldown bar regardless of how many people in the party

Perhaps simply having this done automatically when the anchor is at the bottom could be acceptable too however other people may have other preferences

Second thing is, it would be nice to have the skills such as Magic Level, Melee skill, defensive skill because some creatures will reduce these skills and it would be nice to know without having to keep the skills window open

Thanks for an amazing software!

**Comment by Mytherin** [comment]
> First I was wondering if it would be possible to have the health list HUD have a reverse order option. I also play as a druid and what I mean by this is I like to have the health list near my cooldowns however I would like my EK's name to appear at the bottom and then everyone else in the list would appear in order that they appear in the character health list (as they do now but instead of stacking top to bottom the would stack bottom to top therefore my EK would always be the first one above the cooldown bar regardless of how many people in the party)

To have the reverse order can't you just put the EK at the bottom of the list of names?

**Comment by jeremgod** [comment]
Yes but I have a list of several people I sometimes hunt with a party of 2 sometimes a party of 4 sometimes 5 in the party but since the names stack top to bottom I have to change the Y offset so that the bottom name aligns itself with the cooldown bar based on the amount of people in my party. What I would want is to have the EK first in the list and leave him first in the list, have the anchor at the bottom left and have the names stack bottom to top so that way I can leave EK first in the list and he will always be "first" in my list (first going bottom to top) and regardless of if we're only 2 in a party or if we're 5 there's a lot of free space above the cooldown bar the way I have it setup

**Comment by oufooz** [comment]
I think what he means is .. to make the anchor point for the healthlist . the bottom left corner not top left.

**Comment by jeremgod** [comment]
> I think what he means is .. to make the anchor point for the healthlist . the bottom left corner not top left.

Alright with 2 names and these settings on a 1920x1080 resolution with the tibia window maximized (not full screen) and the game window stretched as much as possible (leaving 4 lines of chat windows at the bottom)

![image](https://cloud.githubusercontent.com/assets/15100569/16171930/6cc00c3e-3552-11e6-86aa-7cb09048e35f.png)

This is how it looks. Perfect right?

![image](https://cloud.githubusercontent.com/assets/15100569/16171937/8ddfeec0-3552-11e6-8cb7-a3918895507c.png)

Look what happens when I want to add a third person to the list let say hypothetical paladin, first in order to have Jeremgod (EK in this example) stay at the bottom I needed to remove him from the bottom of the list, add the name in the middle then add him at the bottom again because there's no simple way to insert a character into the list because pressing "enter" to add new name automatically brings the cursor at the bottom of the list and starts writing at the bottom which makes this new name appear below my ek and overlapping on the cooldown bar

![image](https://cloud.githubusercontent.com/assets/15100569/16171934/79b15d26-3552-11e6-8f81-0c142d69de92.png)

So I need to adjust the offset every time the number of people I am hunting with changes and like I said that is often, sometimes we're 2 sometimes we're 5

So what I would want is when the anchor is bottom left, things would stack from the bottom to the top and have it follow the order of the List as it is doing right now.
